src/install/isolated_install.zig

```diff
@@ -636,8 +135,13 @@ pub fn installIsolatedPackages(
             .trusted_dependencies_mutex = .{},
             .trusted_dependencies_from_update_requests = manager.findTrustedDependenciesFromUpdateRequests(),
             .supported_backend = .init(PackageInstall.supported_method),
+            .active_tasks = if (comptime Environment.ci_assert) try manager.allocator.alloc(std.atomic.Value(bool), store.entries.len) else {},
```

Comment:
```suggestion
            .active_tasks = if (comptime Environment.ci_assert) blk: {
                const slice = try manager.allocator.alloc(std.atomic.Value(bool), store.entries.len);
                @memset(slice, .init(false));
                break :blk slice;
            },
```

Then we can get rid of the separate if statement and `@memset` below.

Reply:
ah its only ci_assert hm

Reply:
updated. didn't add an `AtomicBitset`, but curious @taylordotfish is there a better pattern for asserting a function is only run on one thread at a time?